components/builder-api/src/server/resources/projects.rs

```diff
@@ -205,7 +205,7 @@ fn create_project((req, body): (HttpRequest<AppState>, Json<ProjectCreateReq>))
         vcs_type: "git",
         vcs_data: &vcs_data,
         install_id: body.installation_id as i64,
-        visibility: &origin.default_package_visibility.to_string(),
+        visibility: &origin.default_package_visibility.into(),
```

Comment:
is the into necessary here? The origin model should return PackageVisibility by default

Reply:
Good catch